omod/src/main/java/org/openmrs/module/locationbasedaccess/fragment/controller/field/LocationsFragmentController.java

```diff
@@ -22,6 +23,11 @@ public void controller(FragmentModel model,
         model.addAttribute("activeLocations", activeLocations);
 
         model.addAttribute("selectedLocationUuid", null);
+        String sessionLocationId = session.getAttribute(UiSessionContext.LOCATION_SESSION_ATTRIBUTE).toString();
```

Comment:
What happens when session.getAttribute returns null?

Reply:
Added fix to check for null also.